packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.js

```diff
@@ -32,6 +32,7 @@ const appendKey = (id) => `key={\`${id}-\${i}\`}`
 
 const codeByWidgetType = (Widgets) => ({
   [Widgets.ContentRelationship?.CUSTOM_NAME]: (fieldText, useKey) => `<span>{{ ${fieldText} }} TODO</span>`,
```

Comment:
haha I think this can be done the same way as the two others.

Reply:
Good point I will change this